test/puppetlabs/comidi_test.clj

```diff
@@ -271,14 +326,26 @@
 
     (handler {:uri "/bar/something" :request-method :get})
     (is (= (-> test-atom deref :route-info :route-id) "bar-:bar"))
+    (is (= (-> test-atom deref :route-metadata) route-meta))
+
+    (handler {:uri "/false" :request-method :get})
+    (is (= (-> test-atom deref :route-info :route-id) "false"))
+    (is (= (-> test-atom deref :route-metadata) route-meta))
+
+    (handler {:uri "/true" :request-method :get})
+    (is (= (-> test-atom deref :route-info :route-id) "true"))
+    (is (= (-> test-atom deref :route-metadata) route-meta))
+
+    (handler {:uri "/wat" :request-method :get})
+    (is (= (-> test-atom deref :route-info :route-id) "*"))
```

Comment:
OK.  And the overall motivation for adding this stuff was because booleans are supported in bidi, and not supporting them makes it hard for us to get our schemas right?